autoload/neomake/makers/ft/elixir.vim

```diff
@@ -19,3 +19,11 @@ function! neomake#makers#ft#elixir#credo()
       \ 'errorformat': '[%t] %. %f:%l:%c %m'
       \ }
 endfunction
+
+function! neomake#markers#ft#elixir#dogma()
+    return {
+      \ 'exe': 'mix',
+      \ 'args': ['dogma', '%:p', '--format=flycheck'],
+      \ 'errorformat': '%f:%l:%c: %.: %m'
```

Comment:
`%E` means to mark this line as an error.
So you would use `'errorformat': '%E%f:%l:%c: %.: %m'`.

There are several issues regarding to makers not defining the type correctly, and we might use "error" by default because of this.  But anyway, I think it's better to be more explicit.